docs/migrating.md

```diff
@@ -0,0 +1,301 @@
+# Migrating
+
+## Native Signatures and Imports
+
+This guide provides direction for migrating from custom Glint-specific import paths and signature formats to the
+native imports and standardized signatures for `@ember/component`, `@glimmer/component` and `ember-modifier`.
+
+Note that this guide applies to `@glint/environment-ember-loose`, but `@glint/environment-glimmerx` doesn't yet support
+native imports, pending an upgrade of GlimmerX's own dependencies.
+
+### Background
+
+Prior to version `0.7.4`, Glint required users to import the factories and base classes for components, helpers and
```

Comment:
```suggestion
Prior to version `0.7.4`, Glint required users to import the factories and base classes for components, helpers, and
```

OXFORD

Reply:
BOOOOOOOO

Reply:
Less because I have religious convictions about the Oxford comma and more just because I know I've not used it in several other places in this document, I'm going to leave this as is for internal consistency.

Reply:
That seems reasonable.

Also, did we finally just discover a significant disagreement?!? 😱